lib/helpers/body-parser.js

```diff
@@ -18,6 +18,11 @@ function handleBodyFn(options, bodyFn) {
   options.limit = bytes.parse(options.limit || defaultSizeLimit);
 
   return function (req, res, next) {
+    // If the body is already parsed, by e.g. body-parser, then skip parsing.
+    if (typeof req.body === 'object') {
```

Comment:
According to the [documentation](http://expressjs.com/en/api.html#req.body), `req.body` is `undefined` unless a body-parsing middleware has already populated it. So wouldn't it be better to check if it's not `undefined`?